docs/source/release_notes/release-notes.rst

```diff
@@ -23,6 +23,25 @@ Changelog
 .. <<<<------------------------------------------------------------------------------------------------------------->>>>
 
 =======
+v0.7.0
+=======
+
+.. list-table::
+    :widths: 5 40
+    :header-rows: 1
+
+    *   - Type
+        - Change
+
+    *   - |changed|
```

Comment:
Added a "Breaking" category for now. 

However, I'm not clear on what the difference between "Changed" and "Breaking" is; how do you feel about renaming "Changed" -> "Breaking" instead?